src/components/textarea/_macro-options.md

```diff
@@ -30,6 +31,17 @@
 | charCountOverLimitPlural   | string  | false    | The string displayed when multiple characters have been entered after the limit is reached. Set `{x}` in the string to be replaced with the number, for example “You are {x} characters over the limit". Defaults to "You have exceeded the character limit by {x} characters" |
 | charCountOverLimitSingular | string  | false    | The string displayed when one more character has been entered after the limit is reached. Set `{x}` in the string to be replaced with the number, for example “You are {x} character over the limit". Defaults to "You have exceeded the character limit by {x} character"     |
 
+## WordCheckLimit
+
+| Name                       | Type    | Required | Description                                                                                                                                                                              |
+| -------------------------- | ------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| id                         | string  | true     | The HTML `id` of the component                                                                                                                                                           |
```

Comment:
```suggestion
| id                         | string  | true     | The HTML `id` of the textarea                                                                                                                                                           |
```
Think this should be updated here and in the char one too. Makes it more explicit